src/plugins/auto_batch/auto_batch.cpp

```diff
@@ -32,11 +36,9 @@ Blob::Ptr create_shared_blob_on_top_of_batched_blob(Blob::Ptr batched_blob, size
     auto sizePerBatch = batched_blob->size() / batch_num;
     auto layout = batched_blob->getTensorDesc().getLayout();
     SizeVector dims = batched_blob->getTensorDesc().getDims();
-    // the below code is a placeholder for the WIP (22.1) functionality
-    // that will check the reshaping by the batch is robust (CVS-51744)
-    if (layout == InferenceEngine::Layout::NC || layout == InferenceEngine::Layout::NCDHW ||
-        layout == InferenceEngine::Layout::NCHW || layout == InferenceEngine::Layout::NHWC ||
-        layout == InferenceEngine::Layout::NDHWC) {
+    std::stringstream str;
+    str << layout;
+    if (str.str().find("N") == 0) {
```

Comment:
use `ov::layout::has_batch`

Reply:
tis wouldn't work when the code is called from the old api (e.g. legacy apps)